rep-0003.rst

```diff
@@ -124,6 +124,31 @@ Indigo Igloo (May 2014)
   - releasing, documenting and continuous integration testing only catkin-based packages
 - rosbuild-based packages can still be built from source
 
+Jade Turtle (May 2015 - May 2017)
```

Comment:
We discussed putting this into the REP to explicitly call out the expected EOL date, since otherwise it is ambiguous. We can add it to other entries to make it consistent, but I do not think that is in the scope for this pull request.

The start date is not define or described here either, I can add a section describing both items if you would like.